includes/DataObjects/User.php

```diff
@@ -822,8 +825,15 @@ public function oauthCanCreateAccount()
 		}
 	}
 
+	/**
+	 * @return bool
+	 */
 	public function oauthCanCheckUser()
```

Comment:
On the other hand, we might as well start somewhere.  Made oauthCanCheckUser protected.